functions/__dpaste_gist.fish

```diff
@@ -0,0 +1,26 @@
+function __dpaste_gist
+  set -l __gist_desc $argv[1]
+  set -l __gist_file_name $argv[2]
+  set -l __gist_public $argv[3]
+  set -l __gist_file_content (cat $__gist_file_name)
+  set -l __gist_auth (__dpaste_auth)
+  set -l __dpaste_github_git "https://api.github.com/gists"
```

Comment:
I would name variable `__dpaste_url_gist_github_com` and put it to `init.fish` for consistency.

Reply:
Agreed, I'll move it